docs/content/management.md

```diff
@@ -446,7 +446,31 @@ OPA out-of-band
 
 * `iss`: unused for verification even if present in payload
 
+#### Signature Plugin
 
+OPA supports the option to implement your own bundle signing and verification logic. This will be unnecessary 
+for most and is intended for advanced use cases, such as leveraging key-related services from cloud providers. 
+To implement your own signing and verification logic, you'll need to [extend OPA](../extensions). The following are some snippets 
+to point you in the right direction.
+
+```go
+package main
+
+import "github.com/open-policy-agent/opa/bundle"
+
+func main() {
+  bundle.RegisterSigner("my_signer", &SomeCustomSigner{})
+  bundle.RegisterVerifier("my_signer", &SomeCustomVerifier{})
+}
+```
+
+```sh
+opa build --bundle ./my_policy \
+  --output ./my_policy/bundle.tar.gz  \
+  --signing-key my_key_id \
+  --signing-alg RS256 \
+  --signing-plugin my_signer
+```
```

Comment:
Do you have an example of a custom signer and verifier ? I think it will be helpful to show a simple implementation of `SomeCustomSigner` and `SomeCustomVerifier`.

Reply:
I do, but it's ~250 lines of code, is that OK for the docs?

Reply:
Seems like a good candidate for the [contrib](https://github.com/open-policy-agent/contrib) repo, with a pointer from the docs to that.

Reply:
yea, contrib seems like a good spot.  i'll get a PR over there and update these docs to link